src/Nox.Types.EntityFramework/CountryCode3Converter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace Nox.Types.EntityFramework;
```

Comment:
add this to the namespace and folder  ../Types/CountryCode3

we are starting following this conventions https://regusit.atlassian.net/wiki/spaces/NOX/pages/3153461266/Coding+Conventions+Styles+and+Guidelines